modules/nw-external-dns-operator.adoc

```diff
@@ -50,3 +50,43 @@ $ oc get -n external-dns-operator deployment/external-dns-operator
 NAME                    READY     UP-TO-DATE   AVAILABLE   AGE
 external-dns-operator   1/1       1            1           23h
 ----
+
+== External DNS Operator domain name limitations
+
```

Comment:
I think it's worth adding some context of why the limit is there, something similar to the code repository's README:
```
ExternalDNS Operator uses the TXT registry which implies the usage of the new format and the prefix for the TXT records. This reduces the maximum length of the domain name for the TXT records.
Since the TXT record accompanies every DNS record, there cannot be a DNS record without a corresponding TXT record, therefore the DNS record's domain name gets hit by the same limit.
```

Reply:
Addressed